locales/hr/php-inline.json

```diff
@@ -11,38 +11,38 @@
     "attached": "Ovo polje je već pričvršćeno.",
     "before": "To bi trebao biti datum do :date.",
     "before_or_equal": "To bi trebao biti datum prije ili jednako :date.",
-    "between.array": "This content must have between :min and :max items.",
-    "between.file": "This file must be between :min and :max kilobytes.",
-    "between.numeric": "This value must be between :min and :max.",
-    "between.string": "This string must be between :min and :max characters.",
+    "between.array": "mora sadržavati :min - :max stavki.",
```

Comment:
A sentence must begin with a capital letter, unless the grammar rules of your language suggest otherwise.